magnetis/astro-native
# Fix Svg path in icon components [CONFIA-811]

# What
Fix svg path in some icon components.


# Why
To be rendered correctly.


# How

1. Change Svg Path in `ArrowRightDown` and `ArrowRightUp` components.



# Sample
After: 
ArrowRightUp: 
![Screenshot 2023-02-14 at 11 51 01](https://user-images.githubusercontent.com/35939685/218772918-7989cd55-34ef-4474-a94c-de8094d8036e.png)
ArrowRightDown:
![Screenshot 2023-02-14 at 11 51 47](https://user-images.githubusercontent.com/35939685/218773109-ebe5cada-8ea1-4d18-8251-c5440a5f8311.png)

Before:
ArrowRightUp:
![Screenshot 2023-02-14 at 11 52 36](https://user-images.githubusercontent.com/35939685/218773346-d0e7e285-87ad-490c-b4ca-5d2cb7249080.png)
ArrowRightDown:
![Screenshot 2023-02-14 at 11 53 03](https://user-images.githubusercontent.com/35939685/218773480-726511dc-d8e3-49ec-8cb9-55d183a619d3.png)




# QA

1. Run storybook with `yarn storybook`;
2. Run your preferred emulator with `yarn ios` or `yarn android`;
3. In the storybook webpage, choose `IconButton`;
4. Choose `ArrowRightUp` and `ArrowRightDown` options;
5. See if they're align in the center and their directions are according with their names.








[CONFIA-811](https://produtomagnetis.atlassian.net/browse/CONFIA-811)

[CONFIA-811]: https://produtomagnetis.atlassian.net/browse/CONFIA-811?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
